tests/lost_reset/test

```diff
@@ -41,69 +45,77 @@ for ( $i = 0 ; $i < 10 ; $i++ ) {    # iteration count of 10
     ###
     # tests
     # Start floodping to generate activity
-    seek( $fh_tmp, 0, 0 );
-    system("ping -f 127.0.0.1 >/dev/null 2>&1 & echo \$! >$tmpout");
-    my $ping_pid = <$fh_tmp>;
+    seek( $fh_out, 0, 0 );
+    system("ping -f 127.0.0.1 >/dev/null 2>&1 & echo \$! >$stdout");
+    my $ping_pid = <$fh_out>;
     chomp($ping_pid);
 
     # Add rule to generate audit queue events from floodping
     $result =
       system("auditctl -a exit,always -S all -F pid=$ping_pid >/dev/null 2>&1");
-    ok( $result, 0 );    # Was the rule added successfully?
-    sleep 1;
+    sleep 2;
     kill 'TERM', $ping_pid;
     system("auditctl -d exit,always -S all -F pid=$ping_pid >/dev/null 2>&1");
 
     # Restart the daemon to collect messages in the log
     system("service auditd start >/dev/null 2>&1");
-    sleep 1;
 }
 
-# send the reset lost message (NOTE: requires bash)
-seek( $fh_tmp, 0, 0 );
-$result = system(
-    "echo \$BASHPID\ >$tmpout; exec auditctl --reset-lost >$stdout 2>$stderr");
+sleep 1;
+
+my $status_lost;
```

Comment:
Any reason the $status_lost variable declaration couldn't be put inside the if-block below?

Reply:
From @rgbriggs:
> Debug output at the end of the script, otherwise it would have been
localized.

Fair enough.  I remain not a fan of your debug blocks, but since they are already present for this test I'll look the other way here.